drop-down-terminal@gs-extensions.zzrough.org/extension.js

```diff
@@ -435,9 +435,10 @@ const DropDownTerminalExtension = new Lang.Class({
     _updateWindowGeometry: function() {
         // computes the window geometry except the height
         let panelBox = Main.layoutManager.panelBox;
-        this._windowX = panelBox.x;
+        let legacyTray = Main.legacyTray.actor;
+        this._windowX = panelBox.x + legacyTray.x;
```

Comment:
I just wanted to say that summing `x` coordinates looks weird to me. Does it work correctly only because of the fact that one `x` coordinate is `0` in your case?